src/main/java/executor/command/CommandGetUserBalance.java

```diff
@@ -0,0 +1,32 @@
+package executor.command;
+
+import executor.task.TaskList;
+import ui.Wallet;
+import ui.Ui;
+
+public class CommandGetUserBalance extends Command {
```

Comment:
Isn't this Class already implemented as CommandBalance? Remove if so.

Reply:
This is the command which i did last week